sample.ts

```diff
@@ -0,0 +1,52 @@
+import fc from "fast-check";
+
+/**
+ * Script function to compare two fast-check generators by sampling them.
+ * @param generatorA The first generator to compare.
+ * @param generatorB The second generator to compare.
+ * @param numSamples The number of samples to compare.
+ * @returns An array of result objects containing the sample number, values
+ * generated by each generator, and the match object containing whether the
+ * values match and the mismatched items.
+ */
+export const compareGenerators = (
+  generatorA: fc.Arbitrary<any>,
+  generatorB: fc.Arbitrary<any>,
+  numSamples: number
+) => {
```

Comment:
It'd be nice to see the return type here.

Reply:
Ignore as per https://github.com/stacks-network/rendezvous/pull/69#discussion_r1894582703.